app/assets/javascripts/discourse/app/components/sidebar/section-link-prefix.js

```diff
@@ -0,0 +1,25 @@
+import Component from "@glimmer/component";
+
+export default class extends Component {
+  get prefixValue() {
+    if (!this.args.prefixType && !this.args.prefixValue) {
+      return;
+    }
+
+    switch (this.args.prefixType) {
+      case "span":
+        return this.args.prefixValue
```

Comment:
Could we just say here:

```javascript
if this.args.prefixValue.length === 1 {
   return `#{this.args.prefixValue[0]} 100%`
}
```

Reply:
Though in saying this I kind of violated this same thing here 😅   :

https://github.com/discourse/discourse/blob/a65e0d566f66f55244f43512644b94a72cb39912/app/assets/javascripts/discourse/app/lib/hashtag-types/category.js#L15-L24

Reply:
So it's fine if you just want to leave it